src/checkoutservice/README.md

```diff
@@ -5,3 +5,49 @@ Run the following command to restore dependencies to `vendor/` directory:
 ```sh
 dep ensure --vendor-only
 ```
+
+## OpenTelemetry features
+
+### Emoji Legend
+
+- Completed: :100:
+- Not Present (Yet): :red_circle:
+
+### Traces
+
+- [Instrumentation
+  Libraries](https://opentelemetry.io/docs/concepts/instrumenting-library/):
```

Comment:
Not blocking, but can't really see why this line is split into two lines. It makes it more difficult to read. The same split is repeated in most of the bullets. A single line would be better:
"- [Instrumentation  Libraries](https://opentelemetry.io/docs/concepts/instrumenting-library/):"

Reply:
I believe the split is due to markdown lint rules.

Reply:
Yep on the rules! It’s just raw markdown formatting but doesn’t affect how it looks online 

I use the rewrap & markdownlint vs code extensions to do the heavy lifting.